poly-commitment/src/commitment.rs

```diff
@@ -62,8 +62,8 @@ where
         F: FnMut(A) -> B,
         B: CanonicalDeserialize + CanonicalSerialize,
     {
-        let unshifted = self.unshifted.iter().map(|x| f(*x)).collect();
-        let shifted = self.shifted.map(f);
+        let unshifted = self.unshifted.iter().map(|x| f(x.clone())).collect();
+        let shifted = self.shifted.as_ref().map(|x| f(x.clone()));
```

Comment:
I think you can just write `self.shifted.cloned().map(f)`

Reply:
I'm not able to do this. If I write it this way, `f` gets moved, and so I can't use it for `shifted`.

Reply:
yeah it does :o https://doc.rust-lang.org/std/option/enum.Option.html#method.cloned-1

Reply:
Wouldn't compile for me locally :thinking: Guess it's a recent addition, don't really want to deal with the rust bump in this PR.

Reply:
oh yeah that's a nit so no worries